erpnext/regional/report/fichier_des_ecritures_comptables_[fec]/fichier_des_ecritures_comptables_[fec].py

```diff
@@ -8,24 +8,36 @@
 from frappe import _
 from frappe.utils import format_datetime
 
+COLUMNS = [
```

Comment:
Old style columns, convert them to the new dictionary format?

Reply:
@deepeshgarg007 Done